src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig

```diff
@@ -1,7 +1,7 @@
 <div class="block-exception">
     <div class="block-exception-detected clear-fix">
         <div class="support">
-            <a href="http://symfony.com/support">Need support?</a>
+            <a href="http://symfony.com/support">Support</a> | <a href="https://google.com/search?q={{ exception.message|nl2br|format_file_from_text }}" rel="noopener noreferrer" target="_blank">Google Search</a>
```

Comment:
Indeed, seems like this is a copy/past error. `format_file_from_text` and `nl2br` both generate HTML tags, which surely is something we don't want here.

I don't think tags exists in the error message, so `striptags` doesn't make sense as well.

Btw, I think we need a customized filter. E.g. error messages often show app specific stuff (like file paths, service names, etc.) those should be stripped imo as they will often result in "0 results found" on Google.